packages/observability/setup/README.md

```
@@ -32,89 +30,196 @@ spec:
+}'
+```
+
+Wait for the observability stack pods to come up:
+
+```bash
+oc get pods -n redhat-ods-monitoring -w
+```
+
+Expected pods (all Running):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)